terminal/display_posix.go

```diff
@@ -0,0 +1,11 @@
+// +build !windows
+
+package terminal
+
+import (
+	"fmt"
+)
+
+func EraseInLine(mode EraseLineMode) {
```

Comment:
Sorry confused, you want the func `EraseInLine` renamed to `EraseLine` or the `EraseLineMode` renamed to `EraseLine`?   I think the function name could go either way.  The CSI-n-K operator is apparently called "EL – Erase in Line" so I think that is where the name came from.